led_control:
  acquisition_rate: 10
  subscribe_topic: "/led_frequency"
